### DISCUSSION TITLE
Disable action logging for models without Actionable trait

### DISCUSSION THREAD

**Discussion by mikkellindblom** [laravel/nova-issues#4427] [Ideas & Feature Requests]
- Laravel Version: 9.17.0
- Nova Version: 4.8.1
- PHP Version: 8.0

### Description:

"Update" and "Create" events are saved in the Action log for models without the ```Laravel\Nova\Actions\Actionable``` trait. This behaviour is unexpected in my opinion. Currently there is no way to disable action logging for specific models.

**Comment by vFire** [comment]
Agree

**Comment by crynobone** [comment]
Marking this as "Idea & Feature Request" since it changes the expected behavior of Nova. "Unexpected" as it may it still is the current behavior.
